roles/openshift_metrics/defaults/main.yaml

```diff
@@ -59,3 +59,6 @@ openshift_metrics_cassandra_pvc_access: "{{ openshift_hosted_metrics_storage_acc
 openshift_metrics_hawkular_user_write_access: False
 
 openshift_metrics_heapster_allowed_users: system:master-proxy
+
+openshift_metrics_cassandra_enable_prometheus_endpoint: True
```

Comment:
is there ever a time where we one of these would be true and the other would be false?

Reply:
@ewolinetz sorry, I lost track of this pr, these are separate independent variables, we should have 2.